### PR TITLE
Added Math.trunc to toSecondsFromEpoch to conform the result to u64.

### LINKED ISSUE #4546: Deno.utime() errors when supplied date parameter
Given the following code:
```javascript
await Deno.utime("my_file.txt", new Date(), new Date());
```
The process errors out on v0.38.0 with:

> error: Uncaught InvalidData: invalid type: floating point `1585688574.996`, expected u64

The same error is present with `Deno.utimeSync()` as well.

### PR DESCRIPTION
**Pull request by parkerdgabel**
The conversion to seconds from unix epoch resulted in a float but a u64 is required. I truncated the result to conform it to u64.

Fixes #4546

### CODE REVIEW

**Comment by ry** [comment]
I'm not sure I understand. Is this fixing a bug?

**Comment by parkerdgabel** [comment]
Yes. It is fixing https://github.com/denoland/deno/issues/4546

**Comment by ry** [comment]
@parkerdgabel I see - thanks - could you add a test to `cli/js/tests/utime_test.ts`?

**Comment by parkerdgabel** [comment]
Will do

**Comment by parkerdgabel** [comment]
@ry I've added the tests.

File: cli/js/tests/utime_test.ts [hunk 1]
```diff
@@ -158,6 +183,32 @@ unitTest(
...
+    assertFuzzyTimestampEquals(
+      fileInfo.accessed,
+      Math.trunc(atime.valueOf() / 1000)
+    );
+    assertFuzzyTimestampEquals(
+      fileInfo.modified,
+      Math.trunc(mtime.valueOf() / 1000)
+    );
+  }
+);
```
Comment (ry) [inline cli/js/tests/utime_test.ts:210]: It seems the test is included twice

**Pull request merged**